manifests/params.pp

```diff
@@ -22,4 +26,5 @@
   Optional[String] $aliasesseltype,
   Optional[String] $seltype,
 ) {
+  assert_private()
```

Comment:
This also break compatibility.  However this class is empty so maybe we can delete it completely?

The `foo::param` class predates Hiera and was used to setup os-specific variables. Then the `foo` class could inherit from it to get that default value and allow the user to override it.  This `params` class with parameters is unusual and probably something that even predates that, and we can see direct use of `postfix::params::foo` here and there in the code.

Maybe for now we can ignore this smell and just remove the assert_private to fix CI.

Reply:
CI Tests reworked